cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -111,7 +111,7 @@ spec:
       serviceAccountName: rook-ceph-system
       containers:
       - name: rook-ceph-operator
-        image: rook/ceph:master
+        image: rajasing/ceph:test
```

Comment:
don't forget to revert your test changes

Reply:
yep, It’s still in WIP, I just wanted to get yours reviews.
Thanks